modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7825,22 +7871,40 @@ public void testPatchPutCustomObjectEntryWithDuplicateExternalReferenceCode()
 			Http.Method.PUT, _objectDefinition2, _siteScopedObjectDefinition2);
 	}
 
+	@Test
+	public void testPatchScopeScopeKeyPageWithSiteExternalReferenceCode()
```

Comment:
Could you let me know where you got the name of this method?
The method I saw the name is `patchScopeScopeKeyByExternalReferenceCode` so the name should be `testPatchScopeScopeKeyByExternalReferenceCode`

Reply:
Can you add the three scenarios (ERC, ID, and name)?